lib/telekinesis/consumer/kcl.rb

```diff
@@ -23,6 +23,12 @@ class KCL
       # client in the same `:app` on the same host, make sure you set this to
       # something unique!).
       #
+      # Note: to configure the KCL clients to talk to a local DynamoDB server,
+      # pass in an instance of `AmazonDynamoDBClient` configured to talk to a
+      # local server as the second (optional) parameter to the KCL initialize
+      # method. If not provided, the KCL client will talk to AWS DynamoDB
+      # directly.
+      #
```

Comment:
mind rewording this so it's a little less specific? there are quite a few reasons I can see for passing a DynamoDB client and running a local dynamo doesn't seem like it'll come up that often.

i was thinking something along the lines of 

> Clients interested in configuring their own `AmazonDynamoDB` client may pass an instance as the second argument. If not configured, the client will use a default AWS configuration.